src/components/core/Alert.vue

```diff
@@ -19,12 +19,15 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
   <div v-if="alert">
     <v-alert
       :value="true"
-      :color="alert.getColor()"
+      :type="alert.getColor()"
       :icon="alert.getIcon()"
-      class="ma-0"
+      :class="getColor(alert.getColor())"
       dismissible
+      tile
+      light
+      colored-border
     >
-      <p class="body-1">{{ alert.getText() }}</p>
+      {{ alert.getText() }}
```

Comment:
Shouldn't this text be wrapped, at least in a `<span/>`?

Reply:
The component ends up wrapping the text in some HTML element later I think. I changed after I noticed some padding/margin, and saw that they don't use it in their examples: https://v15.vuetifyjs.com/en/components/alerts